src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue27367.cs

```diff
@@ -0,0 +1,27 @@
+#if TEST_FAILS_ON_CATALYST && TEST_FAILS_ON_WINDOWS
+// On Catalyst, Swipe actions not supported in Appium.
+//Unable to capture a screenshot with the swipe open in windows, as SwipeRightToLeft does not fully open the SwipeItems and instead closes them.
```

Comment:
Sounds like a bug could you confirm and create a new issue? If create a new issue, please add the link to it in the comment.

Reply:
I have a created the [27436](https://github.com/dotnet/maui/issues/27436)  bug report and added the link in the test case.